content/concepts/issued-currencies/issuing-and-operational-addresses.md

```diff
@@ -50,3 +50,24 @@ As with operational addresses, a standby address must have an accounting relatio
 ### Standby Address Compromise
 
 If a standby address is compromised, the consequences are like an operational address being compromised. A malicious actor can steal any balances possessed by the standby address, and the financial institution can change to a new standby address with no action from customers and partners.
+
+
+## See Also
+
+- **Concepts:**
+    - [Accounts](accounts.html)
+    - [Cryptographic Keys](cryptographic-keys.html)
+- **Tutorials:**
+    - [Become an XRP Ledger Gateway](become-an-xrp-ledger-gateway.html)
+    - [Assign a Regular Key Pair](assign-a-regular-key-pair.html)
+    - [Change or Remove a Regular Key Pair](change-or-remove-a-regular-key-pair.html)
+- **References:**
+    - [account_info method][]
+    - [SetRegularKey transaction][]
```

Comment:
These two links aren't links in the html on a local build.

Reply:
I can't reproduce the problem. These are defined by `{% include '_snippets/rippled-api-links.md' %}` at the end of the file. Maybe you're seeing a weird cached file or not building with Dactyl?

Reply:
<img width="386" alt="Screen Shot 2019-08-12 at 8 30 17 PM" src="https://user-images.githubusercontent.com/8494827/62913219-1e92bb00-bd40-11e9-8bc9-f68d64e08762.png">

I built with Dactyl and tried Firefox, Brave, and Chrome (in incognito and normal modes) and hard refreshed several times; still seeing the issue locally.

Reply:
D'oh. You were right. I fetched your fork but didn't pull the latest changes. PBKAC.